bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,274 @@
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-new.html" %}
+
+{% add_lang_files "mozorg/home/index" %}
+
+{% block gtm_page_id %}data-gtm-page-id="Homepage"{% endblock %}
+
+{% block page_image %}{{ static('img/home/page-image.png') }}{% endblock %}
+
+{% block page_title %}
+  {{ _('Internet for people, not profit') }}
+{% endblock %}
+
+{% block page_desc %}
+  {{ _('Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource, open and accessible to all.') }}
+{% endblock %}
+
+{% block body_id %}home-b{% endblock %}
+{% block body_class %}home-variant home-variant-b{% endblock %}
+
+{% block extra_meta %}
+<!-- validates bing webmaster tools -->
+<meta name="msvalidate.01" content="B7B177115A634927D608514DA17B2574">
+<!-- YouTube Verification -->
+<meta name="google-site-verification" content="U9a6gH32vLIykvntaDToj-ytYhlZ1AfAgVEKstixQIE">
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'home-b' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('mozorg-home-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}
+<header class="section masthead" id="masthead">
+  <div class="content">
+
+  {% block tabzilla_tab %}{% endblock %}
+
+  {% block site_header_nav %}
+    <h1 class="masthead-logo">
+      <img src="{{ static('img/home/new/mozilla-wordmark.svg') }}" alt="Mozilla">
+    </h1>
+    {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='nav-download-firefox') }}
+    <nav class="masthead-nav-main" id="nav-main">
+      <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{ _('Menu') }}</span>
+      <ul id="nav-main-menu">
+        <li class="first about-item">
+          <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{ _('About') }}</a>
+        </li>
+        <li class="contribute-item">
+          <a href="{{ url('mozorg.contribute') }}" data-link-type="nav" data-link-name="Participate">{{ _('Participate') }}</a>
+        </li>
+        <li class="products-item">
+          <a href="{{ url('firefox.family.index') }}" data-link-type="nav" data-link-name="Firefox">{{ _('Firefox') }}</a>
+        </li>
+        <li class="donate-item">
+          <a href="{{ donate_url('header') }}" data-link-type="nav" data-link-name="Donate">{{ _('Donate') }}</a>
+        </li>
+      </ul>
+    </nav>
+  {% endblock %}
+
+  {% block site_header_logo %}
+  {% endblock %}
+
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="main wrapper">
+  <section class="section section-intro" id="intro">
+    <div class="content">
+      {# L10n: line break below for visual formatting only #}
+      <h1><span>{{ _('Internet for people, <br>not profit.') }}</span></h1>
```

Comment:
I'm not even sure it's technically valid any longer. They just recently <a href="http://html5doctor.com/computer-says-no-to-html5-document-outline/">decided to kill off the outlining algorithm in HTML 5.1</a> because browsers still hadn't implemented it, and seemingly had no intention to.